config.go

```diff
@@ -526,7 +526,7 @@ func loadConfig() (*config, []string, error) {
 	// Add default wallet port for the active network if there's no port specified
 	cfg.WalletHosts = normalizeAddresses(cfg.WalletHosts, activeNetParams.WalletRPCServerPort)
 
-	if len(cfg.WalletHosts) < 2 {
+	if len(cfg.WalletHosts) < defaultMinServers {
```

Comment:
Per previous comment, I think just explaining why it's hardcoded at 2 makes more sense.